### ISSUE TITLE
module not found when using with next.js

### ISSUE THREAD

**Issue by snimshchikov** [dmotz/trystero#76]
Next.js doesn't like the new version
```
https://nextjs.org/docs/messages/module-not-found

Import trace for requested module:
./node_modules/webrtc-polyfill/index.js
./node_modules/@thaunknown/simple-peer/lite.js
./node_modules/@thaunknown/simple-peer/index.js
./node_modules/trystero/src/peer.js
./node_modules/trystero/src/strategy.js
./node_modules/trystero/src/nostr.js
./node_modules/trystero/src/index.js
./src/app/play/page.tsx
 ⨯ ./node_modules/webrtc-polyfill/lib/RTCPeerConnection.js:1:1
Module not found: Default condition should be last one
```

**Comment by snimshchikov** [comment]
note: I didn't try to use it outside next

**Comment by snimshchikov** [comment]
Fixed in webrtc-polyfill `1.1.7`
